libs/etterlatte-oppgave-model/src/main/kotlin/Oppgave/OppgaveIntern.kt

```diff
@@ -133,6 +134,13 @@ data class VedtakEndringDTO(
     val vedtakHendelse: VedtakHendelse,
 )
 
+data class NyOppgaveDto(
+    val referanse: String,
```

Comment:
Jeg ser ikke helt behovet for å ha en referanse her? Tror det egentlig bare kan fjernes